packages/server/postgres/migrations/1627394777775_addPollsTable.ts

```diff
@@ -0,0 +1,58 @@
+import {MigrationBuilder, ColumnDefinitions} from 'node-pg-migrate'
+import {Client} from 'pg'
+
+import getPgConfig from '../getPgConfig'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  const client = new Client(getPgConfig())
+  await client.connect()
+
+  await client.query(`
+    DO $$
+    BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'PollStatusEnum') THEN
+        CREATE TYPE "PollStatusEnum" AS ENUM (
+          'creating',
+          'active',
+          'editing',
+          'ended'
+        );
+      END IF;
+    END
+    $$;
+
+    CREATE TABLE IF NOT EXISTS "Poll" (
+      "id" VARCHAR(100) PRIMARY KEY NOT NULL,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "deletedAt" TIMESTAMP WITH TIME ZONE DEFAULT NULL, 
+      "createdById" VARCHAR(100) NOT NULL REFERENCES "User"("id"),
+      "discussionId" VARCHAR(100) NOT NULL REFERENCES "Discussion"("id"),
+      "winningOptionId" VARCHAR(100),
```

Comment:
This probably should have `REFERENCES "PollOption"("id")` but the only option to do so is to alter the table after its creation. @mattkrick any thoughts on this?

Reply:
i'd just remove this field for now as it's a 2nd source of truth. 
if we wanted to, we could add `winningOptionId` to the graphql schema & compute it by fetching all the poll options & returning the one with the most voteUserIds

Reply:
The initial idea was to calculate the final result once, when the poll is ended, to avoid querying and calculating that for every user, every time the discussion thread is fetched. I was especially worried if that'd cause some performance issues when there's a lot of people in the meeting. Any thoughts on this @mattkrick? If that's ok, then I'm all in for removing this field! 👍